examples/react-signer/src/App.tsx

```diff
@@ -5,14 +5,16 @@ function App() {
   const { component, state } = useSigner()
 
   return (
-    <div className="flex flex-col justify-center w-[450px] h-[640px] bg-white dark:bg-black">
-      {State.LOADING === state || State.READY == state || !component ? (
-        <div className="flex items-center justify-center">
-          <Loader />
-        </div>
-      ) : (
-        <div className="pt-[58px] pb-[30px] px-[20px] h-full flex flex-col">{component}</div>
-      )}
+    <div className="flex items-center justify-center h-[100vh]">
```

Comment:
seems min-h, user can resize window to be so small, that content will be cut, also seems we need same for width

Reply:
height fixed, as for the width - didn't notice any issues, looks good